it/controllers/FinancialTransactionsComponentSpec.scala

```diff
@@ -18,68 +18,17 @@ package controllers
 
 import config.RegimeKeys
 import helpers.ComponentSpecBase
-import helpers.servicemocks.{DesFinancialDataStub, EISFinancialDataStub}
+import helpers.servicemocks.DesFinancialDataStub
 import models.API1166._
 import models.{FinancialRequestQueryParameters, VatRegime}
 import play.api.http.Status._
 import play.api.libs.json.Json
-import testData.{FinancialData1166, FinancialData1811}
+import testData.FinancialData1166
 
 class FinancialTransactionsComponentSpec extends ComponentSpecBase {
 
   "Sending a request to /financial-transactions/:regime/:identifier (FinancialTransactions controller)" when {
 
-    "the useApi1811 feature switch is enabled" when {
```

Comment:
Why have these tests been removed? I understand we have a TODO at the moment - could we instead tweak the positive test so that it checks for the current `""` response rather than no test at all? That keeps the current test in place, confirms more of the code, and also ensures it will fail again and be fixed during the next subtask. I could easily see it being missed otherwise.

Was the "unsuccessful response" test failing? I would have thought it would be unaffected by your changes as it's not dependent on financial data coming back.